### DISCUSSION TITLE
cloudflared

### DISCUSSION THREAD

**Discussion by Rootalith** [DesktopECHO/Pi-hole-for-Android#48] [General]
hello dear dev i try to run cloudflared and i dwonload cloudflared for armv7 many versions and i get error when i try to enable cloudflared.services 
i need to run DoH is there any idea to run cloudflared 
and thank you for this work

**Comment by DesktopECHO** [comment]
Hi,

[Unbound](https://unbound.docs.nlnetlabs.nl/en/latest/) is already pre-installed and configured to use DNS over TLS, which performs a similar role to cloudflared.  When you deploy the Raspbian image, Pi-hole is installed and configured to use Unbound. 

Default configuration for Unbound is [here](https://github.com/DesktopECHO/Pi-hole-for-Android/blob/main/.pihole.conf)

**Reply by Rootalith** [reply]
thanks for reply and thanks for your work on this porject
